mock/sink_interface_test.go

```diff
@@ -0,0 +1,9 @@
+package mock_test
+
+import (
+	"github.com/lyft/gostats"
+	"github.com/lyft/gostats/mock"
+)
+
+var _ stats.Sink = (*mock.Sink)(nil)
```

Comment:
nit: i think we learned that you can use `&mock.Sink{}` and the compiler will still optimize it away